src/components/toggle/toggle.st.css

```diff
@@ -0,0 +1,63 @@
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named:
+        B2,
+        B6,
+        B7,
+        M7,
+        color_MainText_Background;
+}
+:vars {
+	width: 46px;
+	switchSize: 20px;
+}
+.root {
+	-st-states: checked, disabled;
+}
+.background {
+	width: value(width);
+	border: 1px solid value(B7);
+	padding: 1px;
+	border-radius: 100px;
+	background-color: value(color_MainText_Background);
+	cursor: pointer;
+	user-select: none;
+	transition: background-color 200ms linear,
+}
+.input {
+	display: block;
+	opacity: 0;
```

Comment:
the reason I put `opacity: 0` instead of `visibility: hidden` or `display: none` is that you can not navigate to these kinds of input with tab. Input suppose to be "visible"